zerver/views/invite.py

```diff
@@ -243,11 +257,21 @@ def generate_multiuse_invite_backend(
     if len(streams) and not user_profile.can_subscribe_other_users():
         raise JsonableError(_("You do not have permission to subscribe other users to channels."))
 
+    user_groups: list[NamedUserGroup] = []
+    if group_ids:
+        with transaction.atomic(durable=True):
+            for group_id in group_ids:
+                user_group = access_user_group_for_update(
```

Comment:
This does query in a loop, but we do same for streams. Do we want to have a version to reduce queries here?

Reply:
I don't think we expect these lists to be routinely long enough to be worth it. Might be worth it for the streams version at some point, but it's very low priority; it's a low-throughput endpoint and we're talking dozens of extra milliseconds on it.